File: app/src/main/java/vinova/henry/com/hotfilm/models/TrailerResult.java
package vinova.henry.com.hotfilm.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class TrailerResult {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("results")
    @Expose
    private List<Trailer> trailers = null;

    public Integer getId() {
        return id;
    }

    public List<Trailer> getTrailerList() {
        return trailers;
    }

}

File: app/src/main/java/vinova/henry/com/hotfilm/models/TrailerResult.kt
package vinova.henry.com.hotfilm.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class TrailerResult {
    @SerializedName("id")
    @Expose
    val id: Int? = null
    @SerializedName("results")
    @Expose
    val trailerList: List<Trailer>? = null

}
